oslo/torch/nn/modules/linear.py

```diff
@@ -396,17 +398,27 @@ def __init__(
         super().__init__(
             in_features=int(in_features // self.tesseract_dim),
             out_features=int(out_features // self.tesseract_dim),
```

Comment:
// 자체가 정수로 맞춰주는 operation이므로 int()는 제거해주셔도 될 듯하네요.
```suggestion
            in_features=in_features // self.tesseract_dim,
            out_features=out_features // self.tesseract_dim,
```

Reply:
@jason9693 앗 아직 요 코멘트가 반영 안된 것 같습니다!